composer.json

```diff
@@ -20,34 +20,34 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.4",
```

Comment:
Why?

Reply:
Probably it is our demo server version

Reply:
<img width="1073" alt="Screenshot 2021-03-15 at 10 36 09" src="https://user-images.githubusercontent.com/39232096/111133007-4f6d4e80-857a-11eb-95f6-64ce39893539.png">